README.md

```diff
@@ -3,7 +3,7 @@
 
 This is a Helm repository for charts published by Cribl, Inc.
 
-With the advent of the workergroup and leader Helm charts, we now have a really fast way to deploy an entire distributed Cribl LogStream environment to a Kubernetes cluster.
+With the advent of the Worker Group and Leader Helm charts, we now have a really fast way to deploy an entire distributed Cribl Stream environment to a Kubernetes cluster.
```

Comment:
@bdalpe The lowercase `workergroup` and `leader` are awkward, but they're (part of) those 2 charts' literal names. You might consider rolling back only those 2 changes. Everything else in this PR is a great update, thanks!

Reply:
I realize that now @mQkatz. Thanks! Change reverted (wrapped in back ticks for clarity).

Reply:
Your choice about the backticks...because the literal names are actually `logstream-workergroup` and `logstream-leader`, and you probably don't want to quote the prefixes.

Reply:
True. Ok, it's been reverted to the original.